packages/web/src/ui/views/viewer.js

```diff
@@ -63,39 +63,57 @@ const viewer = (state, i18n) => {
     render = prepareRender(viewerOptions)
     const gestures = require('most-gestures').pointerGestures(el)
 
+    let delta = [0,0];
+    let deltaPan = [0,0];
+    let timer;
     // rotate
     gestures.drags
       .forEach((data) => {
         const ev = data.originalEvents[0]
-        const shiftKey = (ev.shiftKey === true) || (ev.touches && ev.touches.length > 2)
-        if (!shiftKey) {
-          const now = Date.now()
-          const ms = now - lastRotate
-          if (ms > (1000 / rotateRate)) {
-            const delta = [data.delta.x, data.delta.y].map((d) => -d)
-            const updated = orbitControls.rotate({ controls, camera, speed: rotateSpeed }, delta)
-            controls = { ...controls, ...updated.controls }
-            lastRotate = now
-          }
-        }
-      })
-    // pan
-    gestures.drags
-      .forEach((data) => {
-        const ev = data.originalEvents[0]
+        const {x,y} = data.delta;
         const shiftKey = (ev.shiftKey === true) || (ev.touches && ev.touches.length > 2)
         if (shiftKey) {
-          const now = Date.now()
-          const ms = now - lastPan
-          if (ms > (1000 / panRate)) {
-            const delta = [data.delta.x, data.delta.y].map((d) => d)
-            const updated = orbitControls.pan({ controls, camera, speed: panSpeed }, delta)
-            camera.position = updated.camera.position
-            camera.target = updated.camera.target
-            lastPan = now
-          }
+          deltaPan[0] += x;
+          deltaPan[1] += y;          
+        }else{
+          delta[0] -= x;
+          delta[1] -= y;
         }
-      })
+        if(!timer) timer = requestAnimationFrame(doRotatePan);
+    })
+
+    function doRotatePan(){
+      timer = null;
+      if(delta[0] || delta[1]){      
+        const updated = orbitControls.rotate({ controls, camera, speed: rotateSpeed }, delta)
+        delta = [0,0];
+        controls = { ...controls, ...updated.controls }
+      }
+      if(deltaPan[0] || deltaPan[1]){      
+        const updated = orbitControls.pan({ controls, camera, speed: panSpeed }, deltaPan)
+        deltaPan = [0,0];
+        camera.position = updated.camera.position
+        camera.target = updated.camera.target
+      }
+    }
+
+    // pan
```

Comment:
Unless there is a firm plan to re-enable this code on a finite timeline, commented out code should be removed.